packages/reporting/src/stories/0-Welcome.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
We can remove this.

Reply:
Removed from the dashboard package as well.